SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -385,15 +385,12 @@ void USpatialWorkerConnection::OnConnectionFailure()
 {
 	bIsConnected = false;
 
-	if (UGameInstance* GameInstance = Cast<UGameInstance>(GetOuter()))
```

Comment:
Removing this looks like it'll also mean we're not checking that the outer is a `UGameInstance` Are there any edge cases we should be wary of? (Or could the GameInstance ever be null?)

Reply:
As the game instance is owning this object it can never be null. This fixes the shadowing warning previously present

Reply:
Ok cool, in that case seems good 👍